napari/benchmarks/benchmark_labels_layer.py

```diff
@@ -144,19 +144,22 @@ class Labels3DSuite:
 
     param_names = ['n', 'dtype']
     params = ([2**i for i in range(4, 11)], [np.uint8, np.uint32])
-    if 'PR' in os.environ:
-        skip_params = [(2**i,) for i in range(6, 11)]
 
-    def setup(self, n, dtype):
-        if 'CI' in os.environ and n > 512:
-            raise NotImplementedError('Skip on CI (not enough memory)')
+    skip_params = Skipper(
+        func_pr=lambda x: x[0] > 2**6, func_ci=lambda x: x[0] > 2**9
```

Comment:
what is "x"? It would be better if the variable name could be more descriptive... Otherwise, some comments here would be useful, e.g. "skip if image total size is greater than `2**6`" (although I think this is side length, not size?)

Reply:
I have changed Skipper code to pass unpacked tuple and unpack it and use more descriptive names.